src/controllers/notificationController.ts

```diff
@@ -31,7 +37,29 @@ export default class NotificationController {
   > =>
     withValidatedOrValidationError(
       Notification.decode(req.body),
-      this.notificationService.notify
+      async (data: Notification) => {
+        const userHasSessionOrError = await this.sessionStorage.userHasActiveSessions(
```

Comment:
naming: errorOrHasSession (left side = error, booleans start with `has` or `is`)

Reply:
Fixed in ab4bfa2